.cmake-format.yaml

```diff
@@ -0,0 +1,54 @@
+bullet_char: '-'
```

Comment:
Does this file have to be top-level?

Reply:
I doubt that anyone will run it without our script. In the standard configuration it will not work anyway (tabs). So I would prefer to have the config file in a subdir.

Reply:
> I doubt that anyone will run it without our script.

I **do reformat** the code inside TextMate without running the script `reformat-cmake`. While I understand that you do not like files in the top-level directory, you can pretty much ignore this file, just by hiding invisible file in your development tool of choice…

Reply:
Ok, if you need it there, it is okay.